packages/@webex/plugin-meetings/src/meeting/request.ts

```diff
@@ -156,17 +170,25 @@ export default class MeetingRequest extends StatelessWebexPlugin {
       },
     };
 
-    let deviceCapabilities = [];
     if (breakoutsSupported) {
-      deviceCapabilities = [...deviceCapabilities, BREAKOUTS.BREAKOUTS_SUPPORTED];
+      deviceCapabilities.push(BREAKOUTS.BREAKOUTS_SUPPORTED);
     }
     if (liveAnnotationSupported) {
-      deviceCapabilities = [...deviceCapabilities, ANNOTATION.ANNOTATION_ON_SHARE_SUPPORTED];
+      deviceCapabilities.push(ANNOTATION.ANNOTATION_ON_SHARE_SUPPORTED);
     }
-    if (deviceCapabilities.length > 0) {
-      body.deviceCapabilities = deviceCapabilities;
+
+    // Support server side audio disclaimer
+    if (homerDisclaimerEnabled && isJoining) {
+      body.locale = locale || DEFAULT_LOCALE;
+      if (!deviceCapabilities.includes(SERVER_AUDIO_ANNOUNCEMENT_SUPPORTED)) {
```

Comment:
I think deviceCapabilities should not be related with isJoining status

Reply:
Only joining a meeting action need the SERVER_AUDIO_ANNOUNCEMENT_SUPPORTED toggle, not all situations. But this function will be called under many conditions. So it's necessary to add a status to tell them apart. @mickelr

Reply:
And only for the value SERVER_AUDIO_ANNOUNCEMENT_SUPPORTED that the deviceCapabilities prop is related, not for other cases. It doesn't affect other capabilities logics.